### DISCUSSION TITLE
Dataset Mismatch From Actual Description For H2O-PBE0TS

### DISCUSSION THREAD

**Discussion by AnuragKr** [deepmodeling/deepmd-kit#2379] [Q&A] [answered]
Hello,

Dataset Link -- https://www.aissquare.com/datasets/detail?pageType=datasets&name=H2O-PBE0TS

Problem Statement -- On the website and in the paper what dataset size mentioned is actually different from what is present.

System    |||||||||    Original Dataset Size Mentioned    ||||||||||||   Actual Dataset Size Present
1.  lw_pimd     ||||||||   105000  total snapshots    ||||||||||||||                 100000
 
2. ice (b)   |||||||||       24000 total snapshots    ||||||||||||||||                 20000

3. ice (c)   |||||||||       12000 total snapshots   |||||||||||||                      10000

4. ice (d)   |||||||||      12000 total snapshots    ||||||||||||||                      10000

Please check it once and let me know if it is possible to add the missing data.

**Comment by njzjz** [comment, marked as answer]
Please submit to https://github.com/deepmodeling/AIS-Square/

**Reply by AnuragKr** [reply]
Please find clarification of this issue at  -- https://github.com/deepmodeling/AIS-Square/issues/119#event-8847485428
